src/components/form-fields/ApplicationField.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import { InputField } from '../../shared';
+
+export const ApplicationField = () => {
```

Comment:
Are these really generic form fields? I usually consider form fields as generic components that can be reused across different forms.  I think these could just be components/fields specific to the samples form. WDYT?

Reply:
fixed, moved components to their respective folders